worlds/sc2/rules.py

```diff
@@ -480,6 +480,41 @@ def terran_survives_rip_field(self, state: CollectionState) -> bool:
                         and self.terran_sustainable_mech_heal(state))
         )
 
+    def zerg_maw_requirement(self, state: CollectionState) -> bool:
+        """
+        Ability to cross defended gaps, deal with skytoss, and avoid costly losses.
+        :param state:
+        :return:
+        """
+        return (
+                state.has_all({item_names.NYDUS_WORM, item_names.SWARM_QUEEN}, self.player)
+                and (self.morph_brood_lord(state) or self.morph_guardian(state))
+                and (
```

Comment:
No guaranteed air to air unit to guard Guardians/Brood Lords?

Reply:
The Guardians/Brood Lords are literally just for clearing the cannons and prisons; the Zerg essentially _can't_ beat skytoss cost-effectively with air units. Void Rays beat Corruptors, and Mutalisks can't fight in a rip field. If your aerial siege stuff gets attacked, the correct move isn't to defend them, it's to run away.

I'll add a line to ensure they have some form of air-to-air because Zerg players will need all the help they can get here, but it feels like a trap.